patches/minecraft/net/minecraft/client/renderer/ItemBlockRenderTypes.java.patch

```diff
@@ -16,32 +16,42 @@
     private static final Map<Fluid, RenderType> f_109276_ = Util.m_137469_(Maps.newHashMap(), (p_109290_) -> {
        RenderType rendertype = RenderType.m_110466_();
        p_109290_.put(Fluids.f_76192_, rendertype);
-@@ -303,6 +_,7 @@
+@@ -303,6 +_,8 @@
     });
     private static boolean f_109277_;
  
-+   @Deprecated // FORGE: Use canRenderInLayer
++   /** @deprecated Forge: Use {@link net.minecraft.client.resources.model.BakedModel#getRenderTypes(BlockState, net.minecraft.util.RandomSource, net.minecraftforge.client.model.data.ModelData)}. */
++   @Deprecated // Note: this method does NOT support model-based render types
```

Comment:
What are model-based render types?

Reply:
Render types controlled by `BakedModel` instances (the new approach, instead of being managed by `ItemBlockRenderTypes`). The reason it's not supported is because there's no way to faithfully adapt the methods to support multiple render types without partially breaking an implicit contract or causing inconsistent rendering. One of them was adapted in the past, but it had edge cases that would lead to artifacts, as I mentioned in my response to Lex.